src/main/java/com/welovecoding/netbeans/plugin/editorconfig/editor/parser/SyntaxErrorsTask.java

```diff
@@ -0,0 +1,56 @@
+package com.welovecoding.netbeans.plugin.editorconfig.editor.parser;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.swing.text.Document;
+import org.antlr4.SyntaxError;
+import org.netbeans.modules.parsing.spi.ParserResultTask;
+import org.netbeans.modules.parsing.spi.Scheduler;
+import org.netbeans.modules.parsing.spi.SchedulerEvent;
+import org.netbeans.spi.editor.hints.ErrorDescription;
+import org.netbeans.spi.editor.hints.ErrorDescriptionFactory;
+import org.netbeans.spi.editor.hints.HintsController;
+import org.netbeans.spi.editor.hints.Severity;
+
+/**
+ *
+ * @author junichi11
+ */
+public class SyntaxErrorsTask extends ParserResultTask<ECParserResult> {
+
+  @Override
+  public void run(ECParserResult result, SchedulerEvent event) {
+    List<SyntaxError> syntaxErrors = result.getErrors();
+    Document document = result.getSnapshot().getSource().getDocument(false);
+    List<ErrorDescription> errors = new ArrayList<>();
+    for (SyntaxError syntaxError : syntaxErrors) {
+      String message = syntaxError.getMessage();
+      int line = syntaxError.getLine();
+      if (line <= 0) {
+        continue;
+      }
+      ErrorDescription errorDescription = ErrorDescriptionFactory.createErrorDescription(
+              Severity.ERROR,
+              message,
+              document,
+              line);
+      errors.add(errorDescription);
+    }
+    HintsController.setErrors(document, "editorconfig", errors); // NOI18N
```

Comment:
We should make `"editorconfig"` a static variable at a central place to be able to change it everywhere in the code at once if we want.